### ISSUE TITLE
Rename too-long filename

### ISSUE THREAD

**Issue by bpbond** [JGCRI/gcamdata#851]
```
* checking for portable file names ... NOTE
Found the following non-portable file path:
  gcamdata/tests/testthat/comparison_data/gcam-usa/L210.SmthRenewRsrcCurvesGdpElastCapFac_roofPV_USA.csv

Tarballs are only required to store paths of up to 100 bytes and cannot
store those of more than 256 bytes, with restrictions including to 100
bytes for the final component.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
```
Better name nominations @russellhz @pkyle ?

**Comment by pkyle** [comment]
Yikes that one's crazy long! The header ID string is "SmthRenewRsrcCurvesGdpElast", so would "SmthRenewRsrcCurvesGdpElast_roofPV_USA" work? Not sure what the translation is from characters to bytes...could also do SmthRenewRsrcCurvesGdpElast_rPV_USA. Any shorter than that and we should re-name the ID string (and the element of the naming list for this ID string).

**Comment by rplzzz** [comment]
I'm getting an error like that too.  The bogus thing is that these files are in the test directory, which isn't even part of the package per se.  Could we put `tests/testthat/comparison_data` and `tests/testthat/test-data` in .Rbuildignore, or would that somehow prevent them from being used in tests?

**Comment by rplzzz** [comment]
Further discussion:
https://stat.ethz.ch/pipermail/r-package-devel/2015q4/000507.html

**Comment by bpbond** [comment]
>the "tar" file format has been happy with long path names since 2001

Oh jeez.

**Comment by rplzzz** [comment]
Yeah, I was just about to comment on that.  

Given this situation, it's a little annoying that the testthat package burns 10 characters by introducing a completely unnecessary subdir with a long-ish name.

**Comment by rplzzz** [comment]
The developer's response a few messages later is pretty classic too.  It's basically a more diplomatic paraphrasing of "This is stupid.  If you're going to make my life needlessly difficult, then I'll just distribute my software on github."

**Comment by bpbond** [comment]
Yeah, I'd say CRAN has had trouble striking that balance.

**Comment by rplzzz** [comment]
I wouldn't bother fixing it.  It's not actually going to cause a problem for any of our users, and we're already generating notes due to file size anyhow, so we can just resolve to fix all of that if and when we ever feel the need to submit to CRAN.

**Comment by bpbond** [comment]
Well, w're at 102 characters...I might change `comparison_data/` to `cd/` just to get rid of NOTE.

**Comment by bpbond** [comment]
Fixed in #822
